packages/dart_frog_test/README.md

```diff
@@ -0,0 +1,104 @@
+# Dart Frog Test
+
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+[![License: MIT][license_badge]][license_link]
+
+A testing library that makes it easy to test Dart Frog services. It offers helpers to mock requests as well as
+custom matchers in order to write easier to read expectations.
+
+## Installation 💻
+
+**❗ In order to start using Dart Frog Test you must have the [Dart SDK][dart_install_link] installed on your machine.**
+
+Add `dart_frog_test` to your `pubspec.yaml`:
+
+```yaml
+dependencies:
+  dart_frog_test:
+```
+
+Install it:
+
+```sh
+dart pub get
+```
+
+## DartFrogTestContext
+
+This class makes it simple to mock a `RequestContext` for a Dart Frog request handler. To use it, simply import it
+and use its constructor and methods to create the mocker context.
+
+A simple example:
+
+```dart
+// Mocking a get request, which is the default
+import '../../../routes/users/[id].dart' as route;
+
+test('returns ok', () {
+  final context - DartFrogTestContext(
+    path: '/users/1',
+  );
+
+  final response = route.onRequest(context);
+  expect(response.statusCode, equals(200));
+});
+```
+
+If the route handler function reads a [dependency injected via context](https://dartfrog.vgv.dev/docs/basics/dependency-injection), that can also be mocked:
+
+```dart
+// Mocking a get request, which is the default
+import '../../../routes/users/index.dart' as route;
+
+test('returns ok', () {
+  final context - DartFrogTestContext(
+    path: '/users',
+  );
+
+  final userRepository = /** Create Mock */;
+
+  context.provide<UserRepository>(userRepository);
+
+  final response = route.onRequest(context);
+  expect(response.statusCode, equals(200));
+});
+```
+
+Check the `DartFrogTestContext` constructor for all the available context attributes that can be mocked.
```

Comment:
What do you think of linking this to the [API reference hosted by pub.dev](https://pub.dev/documentation/dart_frog_test/latest/)?

Reply:
```suggestion
Check the `DartFrogTestContext` [constructor](https://pub.dev/documentation/dart_frog_test/latest/) for all the available context attributes that can be mocked.
```